docs/streams/quickstart.html

```diff
@@ -130,11 +130,13 @@
     --topic streams-plaintext-input
 Created topic "streams-plaintext-input".
 
+# note: we create the output topic with compaction enabled because the output stream is a KTable
```

Comment:
Users may not know about KTable yet here, maybe we can say "output stream is a changelog stream, see below (ref) for details".